common/server_protocolhandler.cpp

```diff
@@ -765,3 +751,24 @@ void Server_ProtocolHandler::resetIdleTimer()
     lastActionReceived = timeRunning;
     idleClientWarningSent = false;
 }
+
+bool Server_ProtocolHandler::isUserPermittedToJoinRoom(const QString roomPermission, const QString roomPrivilegeLevel)
```

Comment:
This function seems to counteract what's done above slightly. My comment above says that only permission OR roompriv is going to be set... How will a line look if I choose to set room isAdmin only && donator only for example? Should the room say `admin, donator` or something?